<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qsessionframe</class>
 <widget class="QFrame" name="qsessionframe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QLabel" name="sessionMainLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>300</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Session Management Info Page</string>
   </property>
  </widget>
  <widget class="QLabel" name="sessionNameLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>132</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>132</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Session Path</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="sessionNameText">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>420</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>420</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>420</width>
     <height>32</height>
    </size>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="sessionServerUrl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>132</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>132</width>
     <height>18</height>
    </size>
   </property>
   <property name="text">
    <string>Session Server URL</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="sessionUrlText">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>420</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>420</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>32</height>
    </size>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="sessionLogLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>132</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>132</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string>Session Log</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="sessionLogText">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>420</width>
     <height>112</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>420</width>
     <height>112</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>420</width>
     <height>112</height>
    </size>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="detachPushbotton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>100</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>64</height>
    </size>
   </property>
   <property name="text">
    <string>Detach</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="sessionManagerNameText">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>420</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>420</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>420</width>
     <height>32</height>
    </size>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="sessionManagerNameLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>132</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Session Manager</string>
   </property>
  </widget>
  <widget class="QLabel" name="displayNameLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>132</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Session Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="clientIdLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>100</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Client ID</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="displayNameText">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>160</width>
     <height>32</height>
    </rect>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="clientIdText">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>200</y>
     <width>160</width>
     <height>32</height>
    </rect>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="songPathText">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>521</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="songLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Song</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
